src/detect-engine.c

```diff
@@ -2299,7 +2299,15 @@ static int DetectEngineCtxLoadConf(DetectEngineCtx *de_ctx)
             }
 
             if (insp_recursion_limit != NULL) {
-                de_ctx->inspection_recursion_limit = atoi(insp_recursion_limit);
+                if (StringParseInt32(&de_ctx->inspection_recursion_limit,
+                    10, 0, (const char *)insp_recursion_limit) < 0) {
```

Comment:
indent is a bit off here. Can you make the 2nd line of the `StringParseInt32` indent a bit more so it doesn't align exactly with the next statement?